.env_vars.sample

```
@@ -26,6 +26,10 @@ export CONNECT_JWT_ALGORITHM=***
 export ACCOUNT_JWE_PRIVATE_KEY=***
 export IS_JWE_SIGNED=false
 
+## Tracer
+export TRACER_SERVICE_NAME=***
+export LIGHTSTEP_ACCESS_TOKEN=***
```

```suggestion
export TRACER_SERVICE_NAME=
export LIGHTSTEP_ACCESS_TOKEN=
```

especially the second one since, if we copy the sample, it will ends up using the token as `***` is not empty 😅